### PR TITLE
Frontier - Add DHK CW20

### PR DESCRIPTION
**Pull request by UDBlockchain**

### CODE REVIEW

File: osmosis-1/osmosis-frontier.assetlist.json [hunk 1]
```diff
@@ -1395,6 +1395,34 @@
...
+        }
+      ],
+      "base": "ibc/52E12CF5CA2BB903D84F5298B4BFD725D66CAB95E09AA4FC75B2904CA5485FEB",
+      "name": "DHK",
+      "display": "dhk",
+      "symbol": "DHK",
+      "ibc": {
+        "source_channel": "channel-47",
+        "dst_channel": "channel-169",
+        "source_denom": "dhk"
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-frontier.assetlist.json:1419]: Please change source denom to:
"cw20:juno1tdjwrqmnztn2j3sj2ln9xnyps5hs48q3ddwjrz7jpv6mskappjys5czd49"

File: osmosis-1/osmosis-frontier.assetlist.json [hunk 2]
```diff
@@ -1395,6 +1395,34 @@
...
     },
+    {
+      "description": "Building Consensus among Hong Kong People",
+      "type_asset": "cw20",
+      "address": "juno1tdjwrqmnztn2j3sj2ln9xnyps5hs48q3ddwjrz7jpv6mskappjys5czd49",
+      "denom_units": [
+        {
+          "denom": "ibc/52E12CF5CA2BB903D84F5298B4BFD725D66CAB95E09AA4FC75B2904CA5485FEB",
+          "exponent": 0
+        },
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-frontier.assetlist.json:1406]: Please add alias "cw20:juno1tdjwrqmnztn2j3sj2ln9xnyps5hs48q3ddwjrz7jpv6mskappjys5czd49"

File: osmosis-1/osmosis-frontier.assetlist.json [hunk 3]
```diff
@@ -1395,6 +1395,34 @@
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/rac.svg"
       }
     },
+    {
+      "description": "Building Consensus among Hong Kong People",
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-frontier.assetlist.json:1399]: Does this describe the token or does it describe the chain? I don't see any enter in the chain registry to compare it to. Please register this to the Cosmos Chain Registry
https://docs.osmosis.zone/integrate/registration.html#how-to-register-onto-the-cosmos-chain-registry
https://github.com/cosmos/chain-registry

File: osmosis-1/osmosis-frontier.assetlist.json [hunk 4]
```diff
@@ -1395,6 +1395,34 @@
...
+      "name": "DHK",
+      "display": "dhk",
+      "symbol": "DHK",
+      "ibc": {
+        "source_channel": "channel-47",
+        "dst_channel": "channel-169",
+        "source_denom": "dhk"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/dhk.png",
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-frontier.assetlist.json:1422]: These should point to the images uploaded to the Cosmos Chain Registry. Please upload the images there with the registration so we can redirect these. Also, should make sure the images are the same--looks like the png has yellow text while the svg has transparent text.

File: osmosis-1/osmosis-frontier.assetlist.json [hunk 5]
```diff
@@ -1395,6 +1395,34 @@
...
+      "type_asset": "cw20",
+      "address": "juno1tdjwrqmnztn2j3sj2ln9xnyps5hs48q3ddwjrz7jpv6mskappjys5czd49",
+      "denom_units": [
+        {
+          "denom": "ibc/52E12CF5CA2BB903D84F5298B4BFD725D66CAB95E09AA4FC75B2904CA5485FEB",
+          "exponent": 0
+        },
+        {
+          "denom": "dhk",
+          "exponent": 0
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-frontier.assetlist.json:1409]: is this denom meant to be 0? if so, 'dhk' should just be an alias under the first denom units.

File: osmosis-1/osmosis-frontier.assetlist.json (hunk 3 above)
Comment (UDomainBlockchain) [reply osmosis-1/osmosis-frontier.assetlist.json:1399]: This is a DAO token in juno
https://daodao.zone/dao/juno1vntnqevwk47escuxrfrdk99070vs0gczyyx0wa2pp08j25n0ee6qryqhzk

File: osmosis-1/osmosis-frontier.assetlist.json (hunk 5 above)
Comment (UDBlockchain) [reply osmosis-1/osmosis-frontier.assetlist.json]: updated

File: osmosis-1/osmosis-frontier.assetlist.json (hunk 4 above)
Comment (UDBlockchain) [reply osmosis-1/osmosis-frontier.assetlist.json]: updated to url from cosmos chain registry and cosmos registry PR is awaiting as well

File: osmosis-1/osmosis-frontier.assetlist.json (hunk 3 above)
Comment (UDBlockchain) [reply osmosis-1/osmosis-frontier.assetlist.json]: Updated description The DAO token to build consensus among Hong Kong People

File: osmosis-1/osmosis-frontier.assetlist.json (hunk 2 above)
Comment (UDBlockchain) [reply osmosis-1/osmosis-frontier.assetlist.json]: Updated

File: osmosis-1/osmosis-frontier.assetlist.json (hunk 1 above)
Comment (UDBlockchain) [reply osmosis-1/osmosis-frontier.assetlist.json]: Updated

**Comment by UDBlockchain** [comment]
Updated based on your review, thank you

**Pull request merged**